CMakeLists.txt

```diff
@@ -150,6 +152,23 @@ if(ROBOTOLOGY_ENABLE_TELEOPERATION)
   find_or_build_package(walking-teleoperation)
 endif()
 
+# Human Dynamics Estimation
+if(ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS)
+  if (WIN32)
+        set(ROBOTOLOGY_ENABLE_ICUB_HEAD TRUE)
+        set(ROBOTOLOGY_USES_CFW2CAN TRUE)
+        set(ROBOTOLOGY_USES_ESDCAN TRUE)
```

Comment:
I am a bit confused, `CFW2CAN` is not available on Windows. 
More in general, I think that it should be possible to install Human Dynamics Estimation software on Windows even if no iCub devices or ESDCAN devices are installed, so I would avoid to enforce all this option to ON on  Windows. If a user want to enable it on its own machine it can do so, without preventing people that wants to install hde on its own to do so.

Reply:
See `ROBOTOLOGY_USES_CFW2CAN` for an example of an option that can be selected when `ROBOTOLOGY_ENABLE_ICUB_HEAD` is set to true.

Reply:
I do not recall now why I set `ROBOTOLOGY_USES_CFW2CAN ` to `True`. I will check it, and update it.
About the adding options, I will do it, instead of enforcing them to true.
In general, I had in mind that the Windows machine is the producer, then It should be enables all these options. However, it is right, one might use it without some options, e.g. only using Xsens and not the FT shoes.

Reply:
@traversaro  I have updated the dependencies of `wearables` and enabling the options for `human-dynamics` profile, can you check it please again. Thanks

Reply:
for the werables dependency, I add cmake args according to the machine https://github.com/robotology/wearables/blob/42eba1b291ebfda8a27629c4a999cf1225bdca30/CMakeLists.txt#L45-L56